charts/raw/templates/resources.yaml

```diff
@@ -0,0 +1,26 @@
+{{/*
+
+Copyright © 2021 Bedag Informatik AG
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/}}
+{{- if or $.Values.resources $.Values.templates -}}
+  {{- $base := fromYaml (include "raw.labels" .) -}}
```

Comment:
Lets try to eliminate the need to include the library chart, is just for common labels.
Lets implement the commonLabels in this chart and remove the library, or delete this line and remove support for "commonLabels" for now